src/running-a-sora-testnet-node.md

```diff
@@ -166,12 +170,30 @@ If the peer list is empty, your node may not be registered, or there might be ne
 
 ## Perform Transactions via Your Node
 
-### 1. Send and Inspect a Mock Transaction
+### 1. Prepare Your Client
+
+To interact with your node, set up a client.
+Download the `docker-compose.volunteer.client.yml` [configuration file](https://github.com/hyperledger-iroha/iroha/raw/refs/heads/testnet/2.0.0-rc.1/defaults/docker-compose.volunteer.client.yml) and update the following environment variables:
+
+```yml
+# CLIENT CONFIG OVERRIDE
+TORII_URL: <your_host>:8080
+ACCOUNT_PUBLIC_KEY: <your_public_key>
+ACCOUNT_PRIVATE_KEY: <your_private_key>
+```
+
+Next, run the following command to start a container:
+
+```bash
+docker compose -f docker-compose.volunteer.client.yml up -d
+```
+
```

Comment:
I suggest adding example output and pointing out the container name for clarity:
```suggestion

**Example output:**

```log
[+] Running 1/1>docker compose -f docker-compose.volunteer.client.yml up -d
 ✔ Container defaults-clients-1  Started
```
Note the container name: in this example it is `defaults-clients-1`. On your set up it may be different. You need it for the following steps.
```